app/assets/stylesheets/app/dashboard.scss

```diff
@@ -187,6 +189,15 @@
     justify-content: space-evenly;
     color: $primary;
 
+    &--fullscreen {
+      width: 100%;
+      margin: 0;
+    }
+
+    &--fullscreen .matrix__values {
```

Comment:
Merge rule `&--fullscreen .matrix__values` with rule on line 192<br>Selector `matrix__values` should be written in lowercase with hyphens